src/net.cpp

```diff
@@ -64,6 +64,14 @@
 #endif
 #endif
 
+/** Used to pass flags to the Bind() function */
+enum BindFlags {
```

Comment:
Minor nit, but what about switching to C++11 strongly typed enumerations (`enum class`) while we are at it?